Yeah not sure. This isn't exactly helpful:

```
[INFO ]  Performing some trivial sanity checks
[INFO ]  Build started 20200814.144039
[INFO ]  Building environment variables
[EXTRA]  Preparing working directories
[EXTRA]  Installing user-supplied crosstool-NG configuration
[EXTRA]  =================================================================
[EXTRA]  Dumping internal crosstool-NG configuration
[EXTRA]    Building a toolchain for:
[EXTRA]      build  = x86_64-pc-linux-gnu
[EXTRA]      host   = x86_64-pc-linux-gnu
[EXTRA]      target = x86_64-w64-mingw32
[EXTRA]  Dumping internal crosstool-NG configuration: done in 0.03s (at 00:01)
[INFO ]  =================================================================
[INFO ]  Retrieving needed toolchain components' tarballs
[EXTRA]    Retrieving 'mingw-w64-v5.0.1'
[ERROR]   
[ERROR]  >>
[ERROR]  >>  Build failed in step 'Retrieving needed toolchain components' tarballs'
[ERROR]  >>        called in step '(top-level)'
[ERROR]  >>
[ERROR]  >>  Error happened in: do_libc_get[scripts/build/libc/mingw.sh@869]
[ERROR]  >>        called from: main[scripts/crosstool-NG.sh@607]
[ERROR]  >>
[ERROR]  >>  For more info on this error, look at the file: 'build.log'
[ERROR]  >>  There is a list of known issues, some with workarounds, in:
[ERROR]  >>      '/usr/local/ct-ng/share/doc/crosstool-ng/crosstool-ng-1.23.0/B - Known issues.txt'
[ERROR]  >>
[ERROR]  >>  If you feel this is a bug in crosstool-NG, report it at:
[ERROR]  >>      https://github.com/crosstool-ng/crosstool-ng/issues/
[ERROR]  >>
[ERROR]  >>  Make sure your report includes all the information pertinent to this issue.
[ERROR]  >>  Read the bug reporting guidelines here:
[ERROR]  >>      http://crosstool-ng.github.io/support/
[ERROR]   
[ERROR]  (elapsed: 1:42.95)
[01:43] / /usr/local/ct-ng/bin/ct-ng:147: recipe for target 'build' failed

```

--- a/build/builder.sh
+++ b/build/builder.sh
@@ -3,7 +3,7 @@
 set -euo pipefail
 
 image=cockroachdb/builder
-version=20200804-160354
+version=20200818-182851
 
 function init() {
   docker build --tag="${image}" "$(dirname "${0}")/builder"
@@ -180,25 +180,19 @@ set +e
 # is only used if the env var is not set already.
 DATADRIVEN_QUIET_LOG=${DATADRIVEN_QUIET_LOG-true}
 
-# NB: for some mysterious reason, TMPDIR gets lost if passed via --env into
-# docker run below, hence the workaround to invoke `env` inside of the
-# container. See:
-#
-# https://github.com/cockroachdb/cockroach/issues/50507
-# https://github.com/benesch/autouseradd/issues/2
-#
 # shellcheck disable=SC2086
 docker run --init --privileged -i ${tty-} --rm \
   -u "$uid:$gid" \
   ${vols} \
   --workdir="/go/src/github.com/cockroachdb/cockroach" \
+  --env="TMPDIR=/go/src/github.com/cockroachdb/cockroach/artifacts" \
   --env="PAGER=cat" \
   --env="GOTRACEBACK=${GOTRACEBACK-all}" \
   --env="TZ=America/New_York" \
   --env="DATADRIVEN_QUIET_LOG=${DATADRIVEN_QUIET_LOG}" \
   --env=COCKROACH_BUILDER_CCACHE \
   --env=COCKROACH_BUILDER_CCACHE_MAXSIZE \
-  "${image}:${version}" env TMPDIR=/go/src/github.com/cockroachdb/cockroach/artifacts "$@"
+  "${image}:${version}" "$@"
 
 # Build container needs to have at least 4GB of RAM available to compile the project
 # successfully, which is not true in some cases (i.e. Docker for MacOS by default).

--- a/build/builder/Dockerfile
+++ b/build/builder/Dockerfile
@@ -284,8 +284,8 @@ RUN rm -rf /tmp/* /var/lib/apt/lists/*
 
 RUN ln -s /go/src/github.com/cockroachdb/cockroach/build/builder/mkrelease.sh /usr/local/bin/mkrelease
 
-RUN curl -fsSL https://github.com/benesch/autouseradd/releases/download/1.1.0/autouseradd-1.1.0-amd64.tar.gz \
-    | tar xz -C /usr --strip-components 1
+RUN curl -fsSL https://github.com/benesch/autouseradd/releases/download/1.2.0/autouseradd-1.2.0-amd64.tar.gz \
+    | tar xz -C / --strip-components 1
 
 COPY entrypoint.sh /usr/local/bin
 

